docs/source/configuration.rst

```diff
@@ -223,6 +223,35 @@ policy.
    options have been added: ``flood_burst_lines``, ``flood_empty_wait``, and
    ``flood_refill_rate``.
 
+Perform commands on connect
+---------------------------
+
+The bot can be configured to send custom commands upon successful connection to
+the IRC server. This can be used in situations where the bot's built-in
+capabilities are not sufficient, or further automation is desired.
+``$nickname`` can be used in a command as a placeholder, and it will be
+replaced with the bot's nickname, as specified in the configuration
+(:attr:`~CoreSection.nick`).
+
+The list of commands to send is set with
+:attr:`~CoreSection.commands_on_connect`. For example, the following
+configuration::
+
+    [core]
+    commands_on_connect =
+        PRIVMSG X@Channels.undernet.org :LOGIN MyUserName A$_Strong,*pasSWord
+        PRIVMSG IDLEBOT :login $nickname idLEPasswoRD
+
+will, upon connection:
+
+1) identify to Undernet services (``PRIVMSG X@Channels...``)
+2) login with ``IDLEBOT`` using the bot's nickname (``PRIVMSG IDLEBOT ...``)
+
+.. seealso::
+
+   This functionality is analogous to ZNC's ``perform`` module:
+   https://wiki.znc.in/Perform
+
```

Comment:
At some point we'll standardize the indent for blocks like this (I tend to use 4 because I just press <kbd>Tab</kbd> in my editor), but I'm not going to quibble over it until we actually have a documentation style guide. Writing one is on my to-do list for 7.1-ish.